platform/stm32f4_multibots/cmds/i2c/i2c_connect_boards.c

```diff
@@ -0,0 +1,268 @@
+/**
+ * @file
+ * @brief Communicate two boards over I2C
+ *
+ * @date    29.03.2017
+ * @author  Alex Kalmuk
+ */
+
+#include <errno.h>
+#include <string.h>
+#include <stdio.h>
+#include <unistd.h>
+
+#include <kernel/irq.h>
+
+#include "stm32f4_discovery.h"
+
+#define I2C_MODE_MASTER  0
+#define I2C_MODE_SLAVE   1
+
+#define I2C_BUFFERSIZE   256
+
+#define I2C_ADDR         0x30F
+
+static I2C_HandleTypeDef I2cHandle;
+
+static void init_leds(void) {
+	BSP_LED_Init(LED3);
+	BSP_LED_Init(LED4);
+	BSP_LED_Init(LED5);
+	BSP_LED_Init(LED6);
+}
+
+static irq_return_t i2c_ev_irq_handler(unsigned int irq_nr, void *data) {
```

Comment:
Can it be moved to driver folder or somewhere else?

Reply:
Yes, of course, I will take a look at this in more details in the future.